### PR TITLE
Versioning in the release branch for 2.0.0

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @dhalperi @davorbonaci

**Comment by jasonkuster** [comment]
I'm pretty sure that Maven doesn't know about updating the archetype version in the root pom.xml since the release plugin didn't update it on master.

**Comment by dhalperi** [comment]
Starter archetype is not correct. Need to update the reference pom.xml as indicated by the comment in the root pom.xml next to where you made the change.

File: pom.xml [hunk 1]
```diff
@@ -106,15 +106,15 @@
...
     <dataflow.release_name>Google Cloud Dataflow SDK for Java</dataflow.release_name>
-    <dataflow.container_version>${project.version}-20170317</dataflow.container_version>
+    <dataflow.container_version>${project.version}</dataflow.container_version>
     <dataflow.environment_major_version>6</dataflow.environment_major_version>
 
     <!-- Archetype version propagated automatically to underlying archetype modules.
          Example values are: ${project.version}, 1.2.3-SNAPSHOT, [1.0.0,2),
                              [${project.version}, 2), etc.
          Starter archetype reference copy isn't affected by this value; a manual
          change is expected there. -->
```
Comment (dhalperi) [inline pom.xml]: Specifically, this line: manual change expected in starter archetype reference.

Comment (jasonkuster) [inline pom.xml]: This has been fixed.

**Pull request merged**